internal/api/keppel/accounts_test.go

```diff
@@ -1000,6 +1000,54 @@ func TestPutAccountErrorCases(t *testing.T) {
 		ExpectStatus: http.StatusUnprocessableEntity,
 		ExpectBody:   assert.StringData("RBAC policy with \"push\" must also grant \"pull\"\n"),
 	}.Check(t, h)
+	assert.HTTPRequest{
+		Method: "PUT",
+		Path:   "/keppel/v1/accounts/first",
+		Header: map[string]string{"X-Test-Perms": "change:tenant1"},
+		Body: assert.JSONObject{
+			"account": assert.JSONObject{
+				"auth_tenant_id": "tenant1",
+				"rbac_policies": []assert.JSONObject{{
+					"match_cidr": "0.0.0.0/64",
+				}},
+			},
+		},
+		ExpectStatus: http.StatusUnprocessableEntity,
+		ExpectBody:   assert.StringData("\"0.0.0.0/64\" is not a valid cidr\n"),
+	}.Check(t, h)
+	assert.HTTPRequest{
+		Method: "PUT",
+		Path:   "/keppel/v1/accounts/first",
+		Header: map[string]string{"X-Test-Perms": "change:tenant1"},
+		Body: assert.JSONObject{
+			"account": assert.JSONObject{
+				"auth_tenant_id": "tenant1",
+				"rbac_policies": []assert.JSONObject{{
+					"match_repository": "library/.+",
+					"match_username":   "foo",
+					"permissions":      []string{"anonymous_first_pull"},
+				}},
+			},
+		},
+		ExpectStatus: http.StatusUnprocessableEntity,
+		ExpectBody:   assert.StringData("RBAC policy with \"anonymous_pull\" or \"anonymous_first_pull\" may not have the \"match_username\" attribute\n"),
+	}.Check(t, h)
+	assert.HTTPRequest{
+		Method: "PUT",
+		Path:   "/keppel/v1/accounts/first",
+		Header: map[string]string{"X-Test-Perms": "change:tenant1"},
+		Body: assert.JSONObject{
+			"account": assert.JSONObject{
+				"auth_tenant_id": "tenant1",
+				"rbac_policies": []assert.JSONObject{{
+					"match_repository": "library/.+",
+					"permissions":      []string{"anonymous_first_pull"},
+				}},
+			},
+		},
+		ExpectStatus: http.StatusUnprocessableEntity,
+		ExpectBody:   assert.StringData("RBAC policy with \"can_anonymous_first_pull\" may only be for external replica accounts\n"),
+	}.Check(t, h)
```

Comment:
There should be additional tests where:
- an RBAC policy with `anonymous_first_pull` is successfully set and subsequently successfully retrieved, and where
- an RBAC policy with `match_cidr` equal to `0.0.0.0/0` is set and the retrieval does not show this field.